templates/details.html

```diff
@@ -1,113 +1,48 @@
 {% extends "layout.html" %}
 
 {% block content %}
-  <div class="company-page">
-    <div class="company-left-side">
-      <div class="company-detail-title">
-        <img class="detail-logo" src="../static/images/1.jpg"  />
-          <h2 class="company-detail-name">
-            RENAULT
-          </h2>
-      </div>
-      <div class="company-detail-description">
-        <h3 class="company-detail-subtitle">Description</h3>
-        <p class="company-detail-text">
-            This company is a company that company keeps company for companies without company because
-            company is company companion compare compromise such detail much wow vroooom rrratata kabum car
-            braaaa on system hit your enemies with cash, their head under my value shall crash, because "even when
-            i die i have value, at cemeteries i have pockets, so that enemies may see that i take my money with me"
-            This company is a company that company keeps company for companies without company because
-            company is company companion compare compromise such detail much wow vroooom rrratata kabum car
-            braaaa on system hit your enemies with cash, their head under my value shall crash, because "even when
-            i die i have value, at cemeteries i have pockets, so that enemies may see that i take my money with me"
-            This company is a company that company keeps company for companies without company because
-            company is company companion compare compromise such detail much wow vroooom rrratata kabum car
-            braaaa on system hit your enemies with cash, their head under my value shall crash, because "even when
-            i die i have value, at cemeteries i have pockets, so that enemies may see that i take my money with me"
-            This company is a company that company keeps company for companies without company because
-            company is company companion compare compromise such detail much wow vroooom rrratata kabum car
-            braaaa on system hit your enemies with cash, their head under my value shall crash, because "even when
-            i die i have value, at cemeteries i have pockets, so that enemies may see that i take my money with me"
-        </p>
-      </div>
-    </div>
-    <div class="company-right-side">
-      <div class="company-reviews">
-        <div class="company-review-title">
-          <h2>REVIEWS</h2>
-        </div>
-        <div class="review">
-          <h2 class="author">Charlie the fancy</h2>
-          <p class="comment">
-              Such a fancy drive.I am astoundingly pleased with this fancy car.I bought two fancy clios, but my money went adios.
+  <div action="{{ url_for('show_details', company_id=rows.company_id) }}"  class="company-details" >
```

Comment:
You should have `company` instead of `rows`. Please see the comment in the view method.